### PR TITLE
[IMP] payment: payment acquirer whole part should be clickable.

### PR DESCRIPTION
**Pull request by dat-odoo**
Task link : https://www.odoo.com/web?#id=1918459&action=327&model=project.task&view_type=form&menu_id=4720
Pad  link : https://pad.odoo.com/p/r.1fea18f655e99f79eeeb07abc8e9deb2

Purpose: user click anywhere from panel box of payment acquirer to select payment method.
This commit is related to task id 1918459

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/payment/views/payment_templates.xml [hunk 1]
```diff
@@ -103,7 +103,7 @@
                     </t>
                 </t>
                 <t t-foreach="pms" t-as="pm">
-                    <div class="card-body">
+                    <div class="card-body o_payment_acquirer">
```
Comment (JKE-be) [inline addons/payment/views/payment_templates.xml]: o_payment_acquirer
o_delivery_carrier

Could you use more specific name please to avoid potential futur conflict

o_delivery_carrier_select for example

File: addons/payment/static/src/js/payment_form.js [hunk 2]
```diff
@@ -391,6 +392,9 @@ odoo.define('payment.payment_form', function (require) {
 
         // event handler when clicking on a radio button
         radioClickEvent: function (ev) {
+            // radio button checked when we click on entire zone(body) of the payment acquirer
+            this.$('input[type="radio"]:checked').prop("checked", false);
```
Comment (JKE-be) [inline addons/payment/static/src/js/payment_form.js]: idem my other comment about radio button

File: addons/payment/static/src/js/payment_form.js [hunk 3]
```diff
@@ -383,6 +383,7 @@ odoo.define('payment.payment_form', function (require) {
         // event handler when clicking on 'and more' to show more payment icon
         onClickMorePaymentIcon: function (ev) {
             ev.preventDefault();
+            ev.stopPropagation();
```
Comment (JKE-be) [inline addons/payment/static/src/js/payment_form.js]: why this change in this pr ? 

It is really needed, which behaviour is fixed by this line?

Comment (mba-odoo) [reply addons/payment/static/src/js/payment_form.js]: @JKE-be , See the screenshot: https://drive.google.com/file/d/13hyGQ-D9hvOL4DmGUTVzSN5fIl_JsynE/view?usp=drivesdk

Right now, 'Ingenico' is selected, and we want to see more card icons of 'PayPal'. This should not select 'PayPal'. So we need to stop the propagation of the event bound on 'and more' button.

**Comment by JKE-be** [comment]
@mba-odoo 

I just checked on runbot and it is nice.

Still a small bug when you click on the radio itself, it doesn't work
https://youtu.be/fyfKYGDx-zA

Can you check it before to merge ? ty

**Comment by mba-odoo** [comment]
@JKE-be , My bad, done the changes. also,  removed label binding with radio because onclick of the label it's going to trigger event two times.

**Comment by JKE-be** [comment]
Thank you mba

@robodoo r+
